File: ncDados/dTipoEmissao.vb
Namespace nsDados

  Public Class ColecaoTipoEmissao
    Inherits List(Of dTipoEmissao)
  End Class

  Public Class dTipoEmissao

    Private _cid As Integer
    Private _codigo As String
    Private _descricao As String

    Public Property cid() As Integer
      Get
        Return _cid
      End Get
      Set(ByVal value As Integer)
        _cid = value
      End Set
    End Property

    Public Property codigo() As String
      Get
        Return _codigo
      End Get
      Set(ByVal value As String)
        _codigo = value
      End Set
    End Property

    Public Property descricao() As String
      Get
        Return _descricao
      End Get
      Set(ByVal value As String)
        _descricao = value
      End Set
    End Property

  End Class

End Namespace

File: ncModelos/dTipoEmissao.cs
using System;
using System.Collections.Generic;

namespace ncModelos
{
    public class ColecaoTipoEmissao : List<dTipoEmissao>
    {
    }

    public class dTipoEmissao
    {
        private int _cid;
        private string _codigo;
        private string _descricao;

        public int cid
        {
            get => _cid;
            set => _cid = value;
        }

        public string codigo
        {
            get => _codigo;
            set => _codigo = value;
        }

        public string descricao
        {
            get => _descricao;
            set => _descricao = value;
        }
    }

}
